tests/wallet_operation.rs

```diff
@@ -35,27 +38,142 @@ fn using_network(world: &mut TestingWorld, str_chain: String) {
+            let specif_wallet = match type_wallet.as_str() {
+                "stored-huge" => format!("{}/wallet_data_huge", stored_wallets.clone()),
+                "stored-tiny" | _ => {
+                    format!("{}/wallet_data_tiny", stored_wallets.clone())
+                }
+            };
+            let src_wallet = Path::new(&specif_wallet);
+
+            // get wallet_data path
+            let wallet_data = get_wallet_chain_data(&world.chain_type, "wallet");
+
+            // remove wallet_data
+            remove_wallet_chain_path(&world.chain_type, "wallet");
+
+            // copy specifc wallet_data to default dest
+            copy_dir_to(src_wallet, wallet_data.as_path())
+                .expect("Can't copy wallet_data into .epic/network");
+
+            wait_for(5);
 
-    world.passphrase = get_passphrase(&wallet_init);
+            // 2. Chain copy and paste step
+            let specif_chain = format!("{}/chain_data", stored_wallets.clone());
+            let src_chain = Path::new(&specif_chain);
+
+            // get chain_data path
+            let chain_data = get_wallet_chain_data(&world.chain_type, "chain");
+
+            // remove chain_data
+            remove_wallet_chain_path(&world.chain_type, "chain");
+
+            // copy specifc wallet_data to default dest
+            copy_dir_to(src_chain, chain_data.as_path())
+                .expect("Can't copy chain_data into .epic/network");
+
+            wait_for(5);
+
+            // 3. get passphrase from stored wallet
+            let wallet_restore = get_restore_command(
+                &world.chain_type,
+                world.wallet_binary.as_str(),
+                world.password.as_str(),
+            );
+            let passphrase = get_passphrase(&wallet_restore);
+            assert!(passphrase.len() > 0, "Error, passphrase is: {passphrase}");
+            world.passphrase = passphrase;
+            //todo!("Copy and paste the chain_data and wallet_data into .epic/chain_type folder")
+        }
+        "new" => {
+            let wallet_init = create_wallet(
+                &world.chain_type,
+                world.wallet_binary.as_str(),
+                world.password.as_str(),
+            );
+
+            world.passphrase = get_passphrase(&wallet_init);
+        }
+        "passphrase-huge" | "passphrase-tiny" => {
+            todo!(
+                "Copy and paste the chain_data into .epic/chain_type folder
+				Open a file that saves a passphrase of huge/tiny wallet and read.
+				Run recovery process"
+            )
+        }
+        _ => panic!("This {type_wallet} type of wallet has not yet been implemented!"),
+    };
 }
 
-#[then(expr = "I run and save info command")]
-fn info_save(world: &mut TestingWorld) {
-    let info_vec = info_wallet(&world.chain_type, &world.wallet_binary, &world.password);
-    world.info_command = InfoWallet::from(info_vec);
+#[given(expr = "I have a wallet in LMDB")]
```

Comment:
Do you make operations on the LMDB wallet?

Reply:
This test is that verifies that the code will migrate the wallet